_runtime/pushup_support.go

```diff
@@ -94,7 +94,7 @@ func Respond(w http.ResponseWriter, r *http.Request) error {
 	case routeNotFound:
 		return NotFound
```

Comment:
I'm OK with renaming this `ErrNotFound`

Reply:
I'm almost certain that `build.NotFound` referred to this variable, but I wanted to make sure that I was correct that this usage:

```
_runtime/cmd/main.go
114:		if errors.Is(err, build.ErrNotFound) {
```

was in fact correct to change

Reply:
Yes, that is correct.